--- entrypoint.sh.orig-as-of-2024-11-22	2024-11-22 14:54:12
+++ entrypoint.sh	2024-11-22 14:54:12
@@ -166,7 +166,7 @@
                 run_as 'php /var/www/html/occ app:list' | sed -n "/Enabled:/,/Disabled:/p" > /tmp/list_before
             fi
             if [ "$(id -u)" = 0 ]; then
-                rsync_options="-rlDog --chown $user:$group"
+                rsync_options="-rlD"
             else
                 rsync_options="-rlD"
             fi
