--- entrypoint.sh.orig-as-of-2022-12-20	2022-12-20 17:11:36.000000000 +0100
+++ entrypoint.sh	2022-12-20 17:11:36.000000000 +0100
@@ -124,6 +124,7 @@
             rsync_options="-rlD"
         fi
 
+	rsync_options="-rlD"
         # If another process is syncing the html folder, wait for
         # it to be done, then escape initalization.
         # You need to define the NEXTCLOUD_INIT_LOCK environment variable
