--- entrypoint.sh.orig-as-of-2023-01-06	2023-01-06 18:05:30.403781378 +0100
+++ entrypoint.sh	2023-01-06 18:06:02.826767097 +0100
@@ -124,6 +124,7 @@
             rsync_options="-rlD"
         fi
 
+	rsync_options="-rlD"
         # If another process is syncing the html folder, wait for
         # it to be done, then escape initalization.
         # You need to define the NEXTCLOUD_INIT_LOCK environment variable
