--- entrypoint.sh	2021-06-22 10:22:16.321496701 +0200
+++ entrypoint.sh.orig-as-of-2021-06-22	2021-06-22 10:21:54.899136792 +0200
@@ -96,8 +96,11 @@
             echo "Upgrading nextcloud from $installed_version ..."
             run_as 'php /var/www/html/occ app:list' | sed -n "/Enabled:/,/Disabled:/p" > /tmp/list_before
         fi
-        rsync_options="-rlD"
-
+        if [ "$(id -u)" = 0 ]; then
+            rsync_options="-rlDog --chown www-data:root"
+        else
+            rsync_options="-rlD"
+        fi
         rsync $rsync_options --delete --exclude-from=/upgrade.exclude /usr/src/nextcloud/ /var/www/html/
 
         for dir in config data custom_apps themes; do
