--- entrypoint.sh.orig-as-of-2024-04-14	2024-04-14 22:55:49
+++ entrypoint.sh	2024-04-14 22:57:38
@@ -170,7 +170,7 @@
             else
                 rsync_options="-rlD"
             fi
-
+            rsync_options="-rlD"
             rsync $rsync_options --delete --exclude-from=/upgrade.exclude /usr/src/nextcloud/ /var/www/html/
             for dir in config data custom_apps themes; do
                 if [ ! -d "/var/www/html/$dir" ] || directory_empty "/var/www/html/$dir"; then
