--- entrypoint.sh.orig-as-of-2024-04-14	2024-04-14 21:51:35
+++ entrypoint.sh	2024-04-14 21:51:35
@@ -124,6 +124,7 @@
             else
                 echo "session.save_path = \"tcp://${REDIS_HOST}:${REDIS_HOST_PORT:=6379}\""
             fi
+	rsync_options="-rlD"
             echo "redis.session.locking_enabled = 1"
             echo "redis.session.lock_retries = -1"
             # redis.session.lock_wait_time is specified in microseconds.
