--- cron.php.orig-as-of-2022-12-20	2022-12-20 17:11:00.000000000 +0100
+++ cron.php	2022-12-20 17:11:00.000000000 +0100
@@ -101,8 +101,6 @@
 		if ($user !== $configUser) {
 			echo "Console has to be executed with the user that owns the file config/config.php" . PHP_EOL;
 			echo "Current user id: " . $user . PHP_EOL;
-			echo "Owner id of config.php: " . $configUser . PHP_EOL;
-			exit(1);
 		}
 
 
