--- console.php.orig-as-of-2024-04-14	2024-04-14 21:47:16
+++ console.php	2024-04-14 21:47:16
@@ -82,8 +82,6 @@
 			echo "Current user id: " . $user . PHP_EOL;
 			echo "Owner id of the data directory: " . $dataDirectoryUser . PHP_EOL;
 			echo "Try adding 'sudo -u #" . $dataDirectoryUser . "' to the beginning of the command (without the single quotes)" . PHP_EOL;
-			echo "If running with 'docker exec' try adding the option '-u " . $dataDirectoryUser . "' to the docker command (without the single quotes)" . PHP_EOL;
-			exit(1);
 		}
 	}
 
