--- console.php.20.0.5.orig-as-of-2021-01-19	2021-01-19 12:03:27.764360454 +0100
+++ console.php	2021-01-19 12:03:55.163728027 +0100
@@ -72,7 +72,7 @@
 		echo "Owner id of config.php: " . $configUser . PHP_EOL;
 		echo "Try adding 'sudo -u #" . $configUser . "' to the beginning of the command (without the single quotes)" . PHP_EOL;
 		echo "If running with 'docker exec' try adding the option '-u " . $configUser . "' to the docker command (without the single quotes)" . PHP_EOL;
-		exit(1);
+		#exit(1);
 	}
 
 	$oldWorkingDir = getcwd();
