--- console.php.orig-as-of-2024-11-22	2024-11-22 12:20:49
+++ console.php	2024-11-22 12:20:49
@@ -70,8 +70,6 @@
 		echo "Current user id: " . $user . PHP_EOL;
 		echo "Owner id of config.php: " . $configUser . PHP_EOL;
 		echo "Try adding 'sudo -u #" . $configUser . "' to the beginning of the command (without the single quotes)" . PHP_EOL;
-		echo "If running with 'docker exec' try adding the option '-u " . $configUser . "' to the docker command (without the single quotes)" . PHP_EOL;
-		exit(1);
 	}
 
 	$oldWorkingDir = getcwd();
